src/main/java/core/basesyntax/models/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class Rectangle implements Figure {
+    private final String color;
+    private final double width;
+    private final double height;
+
+    public Rectangle(String color, double width, double height) {
+        this.color = color;
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+     public void draw() {

```

Comment:
There's an unnecessary space at the beginning of the `draw` method declaration. Make sure to keep the code style consistent and remove any leading spaces.